restore_volume_from_snapshot.sh

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash -e
+
+if [ -z "$1" ]; then
```

Comment:
Input for the snapshot lineage should be a name instead of `$1`. We could also support taking API href for the snapshot also.

Reply:
If this is an attachment, we can use `$1` but lets add a comment for usage.